### DISCUSSION TITLE
Pricing needs to be a bit clearer

### DISCUSSION THREAD

**Discussion by nahtnam** [supabase/supabase#17150] [General]
I was asking this in Github and I think the pricing page needs to be clearer. My initial question was if I upgrade an org to the Pro plan, can I have unlimited 24/7 supabase instances. It seems like the answer is along the lines of: It would cost $25/month + $10 (minimum) for each extra project for the compute pricing: https://supabase.com/docs/guides/platform/compute-add-ons

**Comment by jakeleventhal** [comment]
It also isn't clear that you have to pay for 24/7 compute costs despite it supposedly being "serverless". The pricing page doesn't even mention the fact that you pay for compute for databases. I created an empty project yesterday, did nothing, and now am charged for 17 hours of compute.
<img width="817" alt="Screenshot 2023-12-14 at 8 45 20 AM" src="https://github.com/supabase/supabase/assets/15133828/01aaf0bd-8c95-4567-bab7-1289d95992e1">

**Reply by jakeleventhal** [reply]
Not clear how this can be considered serverless if you have a dedicated server that you pay a fixed rate 24/7 for.

**Reply by GaryAustin1** [reply]
On the pricing.  The way I understand it is you get a $10 credit each month for your first instance's compute.  So the XS compute time is estimated to be that per month.

**Comment by k-abe-bf** [comment]
I agree with you on this matter.
It should be clearly stated that we will have to pay not only the $25 basic fee, but also the computing price that we were exempted from under the Free plan.

**Reply by GaryAustin1** [reply]
Just to be clear the entry pro plan includes the compute for the month for $25.

**Comment by axelschapmann** [comment]
Did you find a solution to avoid this cost ? Did you move your projects to another tool ?

**Comment by aaa3334** [comment]
I 100% agree - I just looked at signing up here (I have 2 databases) but suddenly something which was free on the free plan I have to pay for ontop of everything else? This pricing is insane....
Should defs be included on the lowest level (as it is with the free plan) then upgrade the instance as required.

I can't think of any other subscription service where you have to pay extra on the paid tier which you got free on the free tier...

**Reply by stevenmunro** [reply]
I ended up switching to something else. I couldn't figure out this compute thing either. had two projects on free, needed another one and I just couldn't.

**Comment by noluyorAbi** [comment]
Pricing still feels pretty unclear and hard to see through tbh. I’m paying for Micro compute hours even when a project gets zero traffic. Ended up paying compute for 3 projects (only ~10% discount on that with the Pro plan), plus the $25 base fee and then another ~$18 for compute on top. Kinda rough ngl

**Reply by GaryAustin1** [reply]
Instances are portions of a server running 24 hours per day as long as project is not removed/paused.  There is no charge for amount of compute and no "spin up" time for a call to the instance.  Basically a Pro ORG is $25 and includes $10 compute credit (enough for a micro instance for 1 month).  Every project after that is CPU cost per hour and if for a month is around $10.  All projects in 1 org share egress/disk/db/realtime metrics.

You can have 2 Free projects still in a different ORG.

**Reply by noluyorAbi** [reply]
Thanks a lot for the clarification.

I think where the confusion came from for me is the wording around “compute hours” it sounds very usage-based, while in reality it’s more like “instance uptime per project”, regardless of whether there’s any traffic. In my case I had 3 Micro projects in a single Pro org, so I effectively paid:

* $25 for the Pro org
* $10 compute credit (covering ~1 Micro)
* ~2 additional Micro instances at ~$10 each → ~+$18 total

I’ll restructure my setup/organizations a bit to work around this pricing model and keep the number of paid instances lower. I just kind of wish it were a bit more usage-based (e.g. tied more to actual traffic/load instead of pure uptime), tbh that would feel more intuitive from a developer perspective.

**Reply by GaryAustin1** [reply]
You could make suggestions to Supabase in supabase/supabase issues on doc wording that might help.  You could also make changes to the wording in a PR that make a note of that or make it clearer.   They likely won't see this post.

Also I'm guessing a user's past experience with AWS EC2 or virtual servers versus other platforms that bill for actual compute cycles would influence level of detail needed.
